### ISSUE TITLE
[Question] How in hybrid mode run application with nvidia card?

### ISSUE THREAD

**Issue by VoidArray** [Askannz/optimus-manager#194]
I'm using hybrid mode.

How explicitly run application with nvidia card?
In bumblebee was optirun/primusrun <app>
And here?

**Comment by pssncp142** [comment]
There is nothing as of yet but you can simply define a function in .zshrc (or whatever shell you use):

`prime () { __NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia "$@" }`

Then try:

`prime glxspheres64`

**Comment by Shivelight** [comment]
You can read the official documentation here:
https://download.nvidia.com/XFree86/Linux-x86_64/435.17/README/primerenderoffload.html

If you are using Arch there is [nvidia-prime](https://www.archlinux.org/packages/extra/any/nvidia-prime/) package in the official repo, it's basically a wrapper to the link above.

TL;DR:
```sh
$ __NV_PRIME_RENDER_OFFLOAD=1 __VK_LAYER_NV_optimus=NVIDIA_only __GLX_VENDOR_LIBRARY_NAME=nvidia glxinfo | grep vendor
```
or with `nvidia-prime`
```sh
$ prime-run glxinfo | grep vendor
```

**Comment by Askannz** [comment]
Duplicate of https://github.com/Askannz/optimus-manager/issues/183
